magento/pwa-studio
# [doc] state management

## Description

Create topic about state management in PWA Studio

## Related Issue

Closes #1720 

## Acceptance 



### Verification Stakeholders

* @sirugh 
* @jimbo 

### Verification Steps

1. Navigate to `pwa-devdocs` directory
2. Run linter on the new topic: `npm run lint src/technologies/basic-concepts/state-management/index.md`
3. Verify no linting errors.
4. Start the HTML preview server: `npm run develop`
5. Navigate to `/technologies/basic-concepts/state-management/`
6. Verify page exists and it is highlighted in table of contents

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
